packages/fast-components-styles-msft/src/toggle/index.ts

```diff
@@ -68,60 +85,60 @@ const styles: ComponentStyles<IToggleClassNameContract, IDesignSystem> = (config
             background: "transparent",
             border: `${toPx(1)} solid`,
             borderColor: (): string => {
-                return config.foregroundColor;
+                return foregroundColor;
             },
             borderRadius: toPx(20),
             appearance: "none",
             cursor: "pointer",
             "@media screen and (-ms-high-contrast:active)": {
                 "&::after, &:checked + span": {
                     background: (): string => {
-                        return config.backgroundColor;
+                        return backgroundColor;
                     }
                 },
             },
             "@media screen and (-ms-high-contrast:black-on-white)": {
                 "&::after, &:checked + span": {
                     background: (): string => {
-                        return config.foregroundColor;
+                        return foregroundColor;
                     }
                 }
             },
             "&:checked": {
                 backgroundColor: (): string => {
-                    return config.brandColor;
+                    return brandColor;
                 },
                 borderColor: (): string => {
-                    return config.brandColor;
+                    return brandColor;
                 },
                 "&:hover": {
                     backgroundColor: (): string => {
-                        return Chroma(config.brandColor).alpha(0.8).css();
+                        return Chroma(brandColor).alpha(0.8).css();
                     },
                     borderColor: (): string => {
-                        return Chroma(config.brandColor).alpha(0.8).css();
+                        return Chroma(brandColor).alpha(0.8).css();
                     }
                 },
                 "&:focus": {
                     borderColor: (): string => {

```

Comment:
none of these need to be functions anymore because the config is already available - you can just assign the value to the key.

Reply:
updated this across all other stylesheet instances (toggle, checkbox, button)